### ISSUE TITLE
Some word entries are getting cut in List View

### ISSUE THREAD

**Issue by DmitryMikhailovich** [Jimex/DictTango-Android#36]
**Describe the bug**
Entries for some words (I think only for those with 2 or more entries) for some dictionaries (at least one) in List View are getting cut (their footer content is blank) but in Sliding View it's displayed OK.

**To Reproduce**
0. You should have 0ALD10 dict to reproduce this.
1. Go to Main of Floating Window
2. Search the word ″wheel″
3. Switch to List View
4. Scroll to the end of the word entry
5. See that some footer content of the entry is cut
![Screenshot_20240531_192944_DictTango](https://github.com/Jimex/DictTango-Android/assets/10853270/365c49cc-0278-4c5f-b33d-5d02ae9b9b70)
6. Switch to Sliding View - it is displayed correctly 
![Screenshot_20240531_194947_DictTango](https://github.com/Jimex/DictTango-Android/assets/10853270/83ed0b13-831a-4d5c-8012-1ab8f46a2a67)

**Expected behavior**
All content should be displayed fully in both Sliding and List Views.

**Screenshots or recordings**
Attached above.

**Android (please complete the following information):**
 - Device: Samsung Galaxy S23 Ultra
 -  Android Version 14

**App Info (please complete the following information):**
 - App Version: 2.2.6.1

**Additional context**
Currently I've discovered this only for a single dictionary, but for many words in it. These words have 2 or more entries in the same dictionary (for noun, verb, etc).

**Comment by Jimex** [comment]
The dark mode has a compatibility issue with list view mode, likely a bug in the Android component. Please use sliding mode to avoid this problem.

**Comment by challenges90** [comment]
I found this bug as well. 

It happens in two dictionaries(Collins 2020 and MWU2020)as far as I found.

Before I saw this issue, I thought it was a compatibility issue with the CSS.

They display normal in normal mode. Only cutted in dark mode.

However, they display normal in Mdict and EUdic in dark mode as well.

![Screenshot_2024-07-02-10-57-22-57_cdd7ad362381255b9279c28ba62b7e43](https://github.com/Jimex/DictTango-Android/assets/5657239/e11b21bc-92f6-4ca0-b5b3-de90e8c1a239)
![Screenshot_2024-07-02-10-57-22-57_cdd7ad362381255b9279c28ba62b7e43](https://github.com/Jimex/DictTango-Android/assets/5657239/db3b392a-b6a5-4ce0-a313-f25545e9ddab)
![Uploading Screenshot_2024-07-02-11-05-42-63_75a5fa3d482633fb16bc28d8c4374029.jpg…]()
